community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,53 @@
+# Best practices for managing onnx issues
+These best practices apply to all onnx ecosystem repos under SIG governance.
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
```

Comment:
Since this document applies to all repos under ONNX, this is not true: some of the repos do use templates already.

The wording is odd - these guidelines should state what is expected.

I'm also not sure the value of linking to templates from this doc. Each repo will have its own templates and they are automatically presented when opening a new Issue (or PR)

Reply:
I removed the justification for adding templates and the current state...